### ISSUE TITLE
Inject client configs into params to be validated by ParamsValidator

### ISSUE THREAD

**Issue by RicardoMiguel** [dwyl/aws-sdk-mock#82]
As a user, I'd like to create a new DynamoDB with global configurations (i.e. TableName).

**Comment by RicardoMiguel** [comment]
I was not aware of 'contributions guidelines' which made me do the work step in first place.
You can find it here: https://github.com/dwyl/aws-sdk-mock/pull/69

**Comment by dankrieger** [comment]
I'd like to second the request for this one.  I have a similar issue where mocking Firehose when the service is instantiated as `new AWS.Firehose({ params: { DeliveryStreamName: 'MyDeliveryStreamName' }});` does not include the DeliveryStreamName when validating parameters and fails validation.

**Comment by marcushill** [comment]
I agree. This feature would be very useful. I would be willing to pick up this work if it has been abandoned.
